pages/architecture/naming.mdx

```diff
@@ -74,6 +74,6 @@ Here are the contract addresses and their links to the respective blockchain exp
 
 - **Sepolia Testnet**:
   
-  - Address: `0x5847d20f9757de24395a7b3b47303684003753858737bf288716855dfb0aaf2`
-  - [View on Sepolia StarkScan](https://testnet.starkscan.co/nft-contract/0x5847d20f9757de24395a7b3b47303684003753858737bf288716855dfb0aaf2)
-  - [View on Sepolia Voyager](https://sepolia.voyager.online/contract/0x5847d20f9757de24395a7b3b47303684003753858737bf288716855dfb0aaf2)
+  - Address: `0x00000154bc2e1af9260b9e66af0e9c46fc757ff893b3ff6a85718a810baf1474`
```

Comment:
this address doesn't exist

Reply:
I believe its the exact one the sepolia starknet ID app is calling.
<img width="1453" alt="Screenshot 2024-09-30 at 15 50 48" src="https://github.com/user-attachments/assets/fae51072-7963-47b2-a4a5-d64662c82ccb">
https://sepolia.voyager.online/contract/0x00000154bc2e1af9260b9e66af0e9c46fc757ff893b3ff6a85718a810baf1474#writeContract